### DISCUSSION TITLE
Cant login

### DISCUSSION THREAD

**Discussion by rkingg-dev** [YOURLS/YOURLS#3531] [General]
I encountered a problem with my server. I noticed that the password is not encrypting, and I can't log in using my username and password. The system displays a "Please log in" message. 
When I attempted to enter an incorrect password, the system returned an "Invalid username or password" error message.

I also dont have idea if the attached image is the problem.

![fadebe7f-62fe-46c4-83c4-5a20b8a64a8a](https://user-images.githubusercontent.com/55124882/222304226-e3836417-e225-471b-bafa-f2d0527a4888.jpg)

**Comment by ehthxC7x9D** [comment]
Assuming user/password combo in `user/config.php` line 65 filled out?   See: https://yourls.org/docs/guide/essentials/credentials

Was it working previously and just stopped, or is this a fresh install?  What version... Any other info you have?

**Comment by rkingg-dev** [comment]
Fresh intalled 1.9.1. no error upon installing.

config line 65 filled out.

also tried chmod 0666 still not working

**Comment by ehthxC7x9D** [comment]
I just did a local fresh install of 1.9.1 and was not able to replicate your issue.  Does refreshing your `user/config.php` file and re-adding a user have any effect?  Are you able to use the browser developer window to watch the POST when you click login? I would be interested to see what the return was.

Some more debugging tips: https://yourls.org/docs/guide/troubleshooting/first-steps

**Comment by rkingg-dev** [comment]
I tested it also in local and it works fine.  weird :(

I see nothing on the "console" developer window while trying to login.

**Comment by ehthxC7x9D** [comment]
In the developer window make sure you are on the 'Network' tab.  Then when you click login look for, and click on the  'POST' link (you may need to scroll up) and make sure the `username` and `password` are not empty in the 'Request` tab on the right.  Then click the 'Response' tab again on the right and let me know the output, if any. 

Screen shot attached noting the developer window tabs.

![debugyourls](https://user-images.githubusercontent.com/1045813/222326228-6e63f9ca-3940-4aac-91ef-65d2bb496842.jpeg)

**Comment by rkingg-dev** [comment]
Sorry for the late reply, here are the screenshots. Thanks for helping

![image](https://user-images.githubusercontent.com/55124882/222338570-5749bebb-f241-494c-89aa-5febc7944183.png)

![image](https://user-images.githubusercontent.com/55124882/222338593-2da8945f-1bcf-489f-8691-cd7bdc235e9e.png)

**Comment by ehthxC7x9D** [comment]
Sorry mate I don't know what else to try without being able to replicate it.  Curious you can get your local copy running but presumably issues in production.   The next thing I would dig in is your httpd logs and see what the access and errors look like for login.  Hopefully someone else in the forum has other ideas.

**Comment by rkingg-dev** [comment]
hello any help?

**Reply by dgw** [reply]
It is impossible for anyone to make even wild guesses without logs, details of the configuration for YOURLS and the HTTPd, etc.

@jarebear6expepjozn6rakjq5iczi3irqwphcvb linked to our suggested troubleshooting steps, which includes checking logs that might show an error on the server side.

**Comment by dummi07** [comment]
Hello, 

I have the same problem.

It worked fine for years but suddenly i cannot login any more. 

Correct User-ID / Password ends in "_Please log in_"

Incorrect Password ends in "_Invalid username or password_"

the developer window shows:

![grafik](https://github.com/YOURLS/YOURLS/assets/173772143/753da2d7-e297-4303-8468-6b24c8560590)

![grafik](https://github.com/YOURLS/YOURLS/assets/173772143/ed94ce72-20f3-4561-be1f-3c39523146ee)

The automatic encryption in user/config does not work 

The Version is  1.9.2

The appache log files doesen't show any errors.

debug shows:

_Connected to database blabla on localhost 
SQL perform: SELECT option_name, option_value FROM yourls_options WHERE 1=1 (0.00011 s)
Check for new version: no_  

thanks for any help

**Comment by dummi07** [comment]
Falls es jemanden interessiert, die Lösung steht hier: https://github.com/YOURLS/YOURLS/issues/1652

Die Cookies sind das Problem, Wenn in der Configdatei www.adresse.xy steht, dann kann man sich nicht mit adresse.xy anmelden.

So simpel, so dämlich. 

Solution found: Take a look at https://github.com/YOURLS/YOURLS/issues/1652

The Problem are the cookies. With or without www - that is the question.
